packages/Tailwind/Ignis.Components.HeadlessUI/Aria/AriaPopupExtensions.cs

```diff
@@ -60,4 +62,15 @@ public static void OnKeyDown(this IAriaPopup popup, KeyboardEventArgs eventArgs)
                 }
         }
     }
+
+    private static readonly object scrollOptions = new { Behavior = "smooth", Block = "nearest" };
+
+    private static async Task ScrollIntoViewAsync(string? id, IJSRuntime? jSRuntime)
```

Comment:
Now you can simply pass the `IAriaPopup`  as a parameter and do 
```csharp
var id = popup.GetId(popup.ActiveDescendant);
if (id == null) return;

await using var element = await popup.JSRuntime.InvokeAsync<IJSObjectReference>("document.getElementById", id);
```

Also as I said please remove the `ConfigureAwait(false)` and instead of disposing manually we use the `using` keyword.